ecosystem/sep-0008.md

```diff
@@ -0,0 +1,48 @@
+
+## Preamble
+
+```
+SEP: 0008
+Title: cancelation
+Author: Tempo 
+Status: Pending
+Created: 2018-05-24
+Version: 0.2
+```
+
+## Simple Summary
+
+Currently Anchors do not have automatic ways to cancel transactions.  Cancelations are useful for countries where the payout system is slow (not real time) or for cash payouts. Typical reasons that clients cancel a transaction are that it is taking too long or they put the wrong recipient name. 
+
+## Specification
+
+
+Endpoint: CANCEL_SERVER/cancel?id=tx_id
+
+Purpose: Cancel a transaction sent to an Anchor.
+
+Method: GET
```

Comment:
This should be `POST` as described in https://github.com/stellar/stellar-protocol/pull/98#issuecomment-391691856